cypress/e2e/cloud/billing.test.ts

```diff
@@ -56,170 +52,154 @@ describe('Billing Page PAYG Users', () => {
-      // The implication here is that there is no Upgrade Now button
-      cy.get('.cf-page-header--fluid').children().should('have.length', 1)
-
-      // PAYG section
-      cy.getByTestID('payg-plan--header').contains('Pay As You Go')
-
-      cy.getByTestID('payg-plan--balance-header').contains('Account Balance')
-      cy.getByTestID('payg-plan--balance-body').contains('10.00')
-
-      cy.getByTestID('payg-plan--updated-header').contains('Last Update')
-      cy.getByTestID('payg-plan--updated-body').contains(
-        `${new Date().toLocaleString('default', {
-          month: 'numeric',
-          day: 'numeric',
-          year: 'numeric',
-          hour: 'numeric',
-          minute: '2-digit',
-        })}`
-      )
-
-      // Invoices Section
-      cy.getByTestID('past-invoices--header').contains('Past Invoices')
-      cy.getByTestID('invoice-history--row').should('have.length', 2)
-
-      cy.getByTestID('invoice-history--name')
-        .last()
-        .contains('December 2020 Invoice')
-      cy.getByTestID('invoice-history--amount').last().contains('$100.00')
-      cy.getByTestID('invoice-history--status').last().contains('unpaid')
-
-      // Sort by date
-      cy.getByTestID('invoice-date--sorter').contains('Invoice Date').click()
-
-      // Should now be the first item
-      cy.getByTestID('invoice-history--name')
-        .first()
-        .contains('December 2020 Invoice')
-
-      // Sort by amount
-      cy.getByTestID('invoice-amount--sorter').contains('Amount').click()
-
-      cy.getByTestID('invoice-history--amount').first().contains('$10.00')
-
-      cy.getByTestID('invoice-status--sorter').contains('Status').click()
-
-      cy.getByTestID('invoice-history--status').first().contains('paid')
-
-      // Payment Method Section
-      cy.getByTestID('payment-method--header').contains('Payment Method')
-      cy.getByTestID('payment-display').contains(
-        'Your current payment card is visa 4242 4242 4242 4242 — Expiring 04/24'
-      )
-      cy.getByTestID('edit-payment--button').click()
-      // TODO: render the zuora form somehow
-      cy.getByTestID('payment-form').should('exist')
-
-      cy.getByTestID('cancel-change--button').click()
-      cy.getByTestID('payment-form').should('not.exist')
-
-      // Contact Information Section
-      cy.getByTestID('billing-contact--header').contains('Contact Information')
-      cy.getByTestID('form-label--First Name').contains('First Name')
-      cy.getByTestID('contact-info--Test').contains('Test')
-      cy.getByTestID('form-label--Last Name').contains('Last Name')
-      cy.getByTestID('contact-info--PAYG').contains('PAYG')
-      cy.getByTestID('form-label--Company Name').contains('Company Name')
-      cy.getByTestID('contact-info--InfluxData').contains('InfluxData')
-      cy.getByTestID('form-label--Country').contains('Country')
-      cy.getByTestID('contact-info--USA').contains('USA')
-      cy.getByTestID('form-label--Physical Address').contains(
```

Comment:
These aren't things I changed: editing these test files just enforces the new line width restrictions from prettier.